test/Functions.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
```

Comment:
need to rename this file to "functions.php" - it should have the exact same name as the file to be tested, including starting with lowercase